app/src/androidTest/java/com/duckduckgo/app/browser/BrowserViewModelTest.kt

```diff
@@ -122,6 +122,24 @@ class BrowserViewModelTest {
         assertEquals("duckduckgo.com", testee.viewState.value!!.url)
     }
 
+    @Test
+    fun whenUrlChangedWithDuckDuckGoUrlContainingQueryThenUrlRewrittenToContainQuery() {
+        testee.urlChanged("http://duckduckgo.com?q=test")
+        assertEquals("test", testee.viewState.value!!.url)
```

Comment:
Feels like "url" is the wrong name for this now.  Perhaps it should be "addressBarText" instead?  Likewise urlInput in the BrowserActivity is also for more than just urls (ie search queries) so maybe should be called "addressBarTextInput"? 

@subsymbolic what do you think?

Reply:
Good point

Reply:
Sounds good to me too. We call it the omnibar so how about `omnibarText` and `omnibarTextInput`?